indico/modules/events/registration/templates/management/_registration_details.html

```diff
@@ -25,6 +25,7 @@
         {% if registration.state.name not in ('rejected', 'withdrawn') %}
             {{ _render_withdraw_actions(registration) }}
         {% endif %}
+        {{ _render_allow_modification(registration) }}
```

Comment:
Since it makes no sense to show this if a registration is rejected/withdrawn, I would not show this box in such cases:

```suggestion
        {{ _render_allow_modification(registration) if registration.is_active }}
```